pages/common/dotnet-test.md

```diff
@@ -0,0 +1,16 @@
+# dotnet test
+
+> Execute tests for a .NET application.
+> More information: <https://learn.microsoft.com/dotnet/core/tools/dotnet-test>.
+
+- Execute tests for a .NET project / solution in the current directory:
```

Comment:
```suggestion
- Execute tests for a .NET project/solution in the current directory:
```
We generally don't add spaces between slashes in descriptions (so we could do the same here to stay consistent with other pages having similar syntax).

Reply:
Thank you for the review, I wasn't sure about this, so it's now clear :).